klippa/src/name.rs

```diff
@@ -0,0 +1,123 @@
+//! impl subset() for name table
+
+use std::cmp::Ordering;
+
+use crate::{
+    Plan,
+    SubsetError::{self, SubsetTableError},
+    SubsetFlags,
+};
+
+use write_fonts::{
+    read::{tables::name::Name, FontRef, TableProvider, TopLevelTable},
+    FontBuilder,
+};
+
+// reference: subset() for name table in harfbuzz
+// https://github.com/harfbuzz/harfbuzz/blob/main/src/OT/name/name.hh#L387
```

Comment:
Unstable ref, use a permalink, e.g. https://github.com/harfbuzz/harfbuzz/blob/a070f9ebbe88dc71b248af9731dd49ec93f4e6e6/src/OT/name/name.hh#L387

You can get it from the ... menu in GH